cli/core/pkg/pluginmanager/manager.go

```diff
@@ -171,32 +225,90 @@ func AvailablePluginsFromLocalSource(localPath string) ([]plugin.Discovered, err
 	if err != nil {
 		log.Warningf("Unable to discover standalone plugins from local source, %v", err.Error())
 	}
-	return availablePlugins("", []plugin.Discovered{}, localStandalonePlugins)
+	return availablePlugins([]plugin.Discovered{}, localStandalonePlugins)
 }
 
-func availablePlugins(serverName string, discoveredServerPlugins, discoveredStandalonePlugins []plugin.Discovered) ([]plugin.Discovered, error) {
-	installedSeverPluginDesc, installedStandalonePluginDesc, err := InstalledPlugins(serverName)
+func availablePlugins(discoveredServerPlugins, discoveredStandalonePlugins []plugin.Discovered) ([]plugin.Discovered, error) {
+	installedServerPluginDesc, installedStandalonePluginDesc, err := InstalledPlugins()
 	if err != nil {
 		return nil, err
 	}
 
 	availablePlugins := availablePluginsFromStandaloneAndServerPlugins(discoveredServerPlugins, discoveredStandalonePlugins)
 
-	setAvailablePluginsStatus(availablePlugins, installedSeverPluginDesc)
+	setAvailablePluginsStatus(availablePlugins, installedServerPluginDesc)
 	setAvailablePluginsStatus(availablePlugins, installedStandalonePluginDesc)
 
 	installedButNotDiscoveredPlugins := getInstalledButNotDiscoveredStandalonePlugins(availablePlugins, installedStandalonePluginDesc)
 	availablePlugins = append(availablePlugins, installedButNotDiscoveredPlugins...)
 
+	availablePlugins = combineDuplicatePlugins(availablePlugins)
```

Comment:
non-blocking: It's for a corner case, but we should consider adding a TODO comment to deal with this situation:

if there are multiple discovered (but not installed) plugins of the same name then we will always end up keeping one deterministically (due to the sequence of sources/plugins that we process), but the choice is entirely arbitrary.